src/main/java/org/broadinstitute/dsm/util/externalShipper/GBFRequestUtil.java

```diff
@@ -82,8 +82,8 @@
 
     public final String EXTERNAL_SHIPPER_NAME = "gbf";
 
-    private static int additionalAttempts = 1;
-    private static int sleepInMs = 500;
+    private static final int MAX_ORDER_RETRIES = 5;
+    private static final int GBF_SLEEP_BETWEEN_RETRIES = 7_000;
```

Comment:
500ms is probably too short to overcome a brief network issue or GBF outage.  I upped this to 5 retries and 7 seconds to be on the safe side.